code/__DEFINES/medical.dm

```diff
@@ -0,0 +1,31 @@
+/// Physical statuses
+#define PHYSICAL_ACTIVE "Active"
+#define PHYSICAL_PHYSICALLY_UNFIT "Physically Unfit"
+#define PHYSICAL_UNCONSCIOUS "*Unconscious*"
+#define PHYSICAL_DECEASED "*Deceased*"
+#define PHYSICAL_CANCEL "Cancel"
+
+/// List of available physical statuses
+#define PHYSICAL_STATUSES(...) list(\
+	PHYSICAL_ACTIVE, \
+	PHYSICAL_PHYSICALLY_UNFIT, \
+	PHYSICAL_UNCONSCIOUS, \
+	PHYSICAL_DECEASED, \
+	PHYSICAL_CANCEL, \
+)
+
+/// Mental statuses
+#define MENTAL_STABLE "Stable"
+#define MENTAL_WATCH "*Watch*"
+#define MENTAL_UNSTABLE "*Unstable*"
+#define MENTAL_INSANE "*Insane*"
+#define MENTAL_CANCEL "Cancel"
+
+/// List of available mental statuses
+#define MENTAL_STATUSES(...) list(\
+	MENTAL_STABLE, \
+	MENTAL_WATCH, \
+	MENTAL_UNSTABLE, \
+	MENTAL_INSANE, \
+	MENTAL_CANCEL, \
+)
```

Comment:
Are these lists used outside of `code/modules/mob/living/carbon/human/human.dm`? If not, move them into that file as a local define, putting these above where you actually use them, and then `#undef`ing them afterwards. saves on compiler stuff and is more polite.

it's okay to keep it in the global defines folder if we use them elsewhere, but i noticed we aren't really using `MENTAL_STATUSES` or `PHYSICAL_STATUSES` elsewhere really... the individual defines are probably fine IMO but if we're just using the lists to pass it into tgui_input_list then we should move it just in the scope that we use it for.

Reply:
Thanks for elaborating on that, I'll double-check they're used in multiple places before I finalize the commits. I'm currently attempting to adjust the Medical Records Console UI (TGUI screen) slightly, to match how security crime statuses are implemented. So I think these will eventually be used in a few more places than just `human.dm`.

This current code treats Physical/Mental statuses as editable strings in the Record Console, so any string can be entered, whereas the Med-HUD only allows for options from a fixed list (e.g. Active, Unconscious, Deceased, etc). I think it would make more sense for it to be a series of button selections corresponding to each of these pre-defined statuses. Security consoles have already implemented this, but for warrant statuses, so I'm just reverse engineering that code to fit here.

If that seems outside the scope of this issue, let me know and I'll merge my changes sooner, otherwise I'll see if I can cleanly implement it, so there's consistency among the different records consoles.

Reply:
yeah that's fine my only concern is the two lists `PHYSICAL_STATUSES` and `MENTAL_STATUSES`. i don't really mind having the ten string macros since those are potentially re-usable and extendable to some extent (could have a psychologist hud that points out someone who's been marked as insane)- it just seems that those two lists are only used for passing into the `tgui_input_list`. if you think they can be extendable in some other way, they're okay to keep here- but if there's no reason to do it like that, just move it such that it only exists in that file like:

```dm
// the very top of the file
#define PHYSICAL_STATUSES list("big","chungus","swag")

// insert all of the code that normally exists in that file here
tgui_input_list(..., MENTAL_STATUSES, ...)
// more code afterwards

// the very bottom of the file
#undef PHYSICAL_STATUSES
```

again you don't need to do this if the lists could potentially be re-used elsewhere, but there's no reason to have them outside of the scope of the file if it doesn't really work and could only be used in that medhud "pick a string" context.